src/main/java/gui/CreationModalView.java

```diff
@@ -0,0 +1,215 @@
+package gui;
+
+import javafx.event.EventHandler;
+import javafx.scene.control.Button;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Text;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class responsible for displaying a modal view for the creation of shots.
+ * @author alex
+ */
+public class CreationModalView extends ModalView {
+
+    private static final int width = 600;
```

Comment:
Could you make these more flexible (percentage of scene dimensions or something)?